lib/payment_processor/clients/braintree/subscription_plan_selector.rb

```diff
@@ -3,6 +3,8 @@ module Clients
     module Braintree
 
       class SubscriptionPlanSelector
+        include ActsLikeSelectorWithCurrency
+
         # I know, I know! This is a temporary measure.
```

Comment:
Ok but I meant you could just add them as-is into production.yml for now :) I didn't mean you'd have to add all the ENVs.